### ISSUE TITLE
Unused entities blank on Safari, iOS browsers, and companion app

### ISSUE THREAD

**Issue by n8henrie** [home-assistant/frontend#7747]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

When I go to the "Unused entities" page, the page is blank. It seems to display them for the briefest imaginable flash before they disappear, but it's so quick it's hard to be sure.

This happens in the iOS companion app, Safari iOS, and Firefox iOS. No change with clearing cache, disable ad blocker, or "request desktop version."

iOS 14.2, started happening on 0.117 and continues on 0.118.1. 

Other stuff seems to be working fine. Unused entities working fine on desktop browsers. 

## Expected behavior

Unused entities display as per usual. 

## Steps to reproduce

- `...` in top right corner -> edit dashboard
- `...` in top right corner -> unused entities

## Environment

Noted above 

- Home Assistant release with the issue:
- Last working Home Assistant release (if known):
- Browser and browser version:
- Operating system:

## State of relevant entities

None relevant 

```yaml

```

## Problem-relevant configuration

None, no config changes since appearance of the issue in 0.117.

```yaml

```

## Javascript errors shown in your browser console/inspector

Hard to tell on iOS.

```txt

```

## Additional information 

![image](https://user-images.githubusercontent.com/1234956/99881793-1bd87400-2bd9-11eb-9763-ea7bcdcfdba6.png)

**Comment by melyux** [comment]
It stopped working for me on Safari on Mac too.

**Comment by n8henrie** [comment]
Good call -- I always use Firefox on desktop (still working there), but just check Safari on MacOS and also broken there. Much easier to see the dev tools! No errors in the console.

MacOS 11.0.1
Safari Version 14.0.1 (16610.2.11.51.8)

**Comment by squirtbrnr** [comment]
This still happens with the latest Mac and iOS App Store version of the companion app and HA 2021.3.4. The unused entities list tries to load then just shows up blank. Works fine in Google Chrome on my Mac, no alternative on iOS.

**Comment by melyux** [comment]
I haven't been able to edit my UI in months because of this.

**Comment by melyux** [comment]
Fixed in the iOS app, but not Safari on Mac.

**Comment by spacegaier** [comment]
> 
> 
> Fixed in the iOS app, but not Safari on Mac.

Can you please check the Safari browser console/ log if there are any errors or warnings in there.

**Comment by melyux** [comment]
It's working now on Safari on both Mac and iOS. But it still doesn't work on the Mac Home Assistant companion app. And there's no console for me to look at there.

**Comment by squirtbrnr** [comment]
@melyux what macOS version are you running?  I just updated to macOS Monterey (12.0) about a month ago and have not had any issues loading the unused entities list since then.  Previously I was running macOS Catalina (10.15) and had issues not just with the unused entities list not loading, but other app stability problems.  I skipped macOS Big Sur (11) so I don't know if the problems exist there or not.
